docs/install.rst

```diff
@@ -117,45 +124,47 @@ the remote servers. ::
 
     ./securedrop-admin install
 
-You will be prompted to enter the sudo passphrase for the *Application* and
-*Monitor Servers* (which should be the same).
+You will be prompted to enter the sudo passphrase for the *Application Server* and
+*Monitor Server* (which should be the same).
 
-The install process will take some time, and it will return
-the terminal to you when it is complete.
+The installation process will take some time. It will return you to the
+terminal prompt when complete.
 
-If an error occurs while
-running the install, please check all of the details of the error output.
+If an error occurs while running the install, please save the error output
+in its entirety for reference and troubleshooting.
```

Comment:
what about "save and inspect"? The original wording did not recommend to save the output and this is useful advice. But it would be good to keep the encouragement to look at the output and maybe detect the simpler errors.

Reply:
I agree. I've tried another wording. Let me know if it works!